shido/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "shido",
+  "assets": [
+    {
+      "description": "The native EVM and Wasm, governance and staking token of the Shido Chain",
+      "denom_units": [
+        {
+          "denom": "shido",
+          "exponent": 18
+        }
+      ],
+      "base": "shido",
```

Comment:
base unit must be 0 exponent. Please define another denom unit with 0 exponent above the unit 'shido'.

Reply:
There is still ambiguity about which unit one is referring to because the denom 'shido' for each unit is the same. Please set the denom for the base 0 unit to whatever the denomination onchain is. And then the denom value for the exponent 18 unit must be something different.

e.g., 

```
"denom_units": [
        {
          "denom": "ashido",
          "exponent": 0
        },
        {
          "denom": "shido",
          "exponent": 18
        }
      ],
```
 but don't just copy that example, actually make sure that the 0 exponent denom is what's using onchain.
 Once determined, that is the denom value to be used under `staking` and `fees` in the chain.json file.